cheatsheets/Choosing_and_Using_Security_Questions_Cheat_Sheet.md

```diff
@@ -1,119 +1,149 @@
+Answers should also be checked against a blacklist, including:
 
-### Step 5) Periodically Have Your Users Review their Questions
+- The username or email address.
+- The user's current password.
+- Common strings such as "123" or "password".
 
-Many companies often ask their users to update their user profiles to make sure contact information such as email addresses, street address, etc. is still up-to-date. Use that opportunity to have your users review their security questions. (Hopefully, at that time, they will be in a bit less of a rush, and may use the opportunity to select better questions.) If you had chosen to encrypt rather than hash their answers, you can also display their corresponding security answers at that time.
+### Renewing Security Questions
 
-If you keep statistics on how many times the respective questions has been posed to someone as part of a Forgot Password flow (recommended), it would be advisable to also display that information. (For instance, if against your advice, they created a question such as "What is my favorite hobby?" and see that it had been presented 113 times and they think they might have only reset their password 5 times, it would probably be advisable to change that security question and probably their password as well.)
+If the security questions are not used as part of the main authentication process, then consider periodically prompting the user to review their security questions and verify that they still know the answers. This should give them a chance to update any answers that may have changed (although ideally this shouldn't happen with good questions), and increases the likelihood that they will remember them if they ever need to recover their account.
 
-### Step 6) Authenticate Requests to Change Questions
+## System Defined Security Questions
 
-Many web sites properly authenticate change password requests simply by requesting the current password along with the desired new password. If the user cannot provide the correct current password, the request to change the password is ignored. The same authentication control should be in place when changing security questions. The user should be required to provide the correct password along with their new security questions & answers. If the user cannot provide the correct password, then the request to change the security questions should be ignored. This control prevents both Cross-Site Request Forgery attacks, as well as changes made by attackers who have taken control over a users workstation or authenticated application session.
+System defined security questions are based on information that is already known about the user. The users' personal details are often used, including the full name, address and date of birth - however these can easily be obtained by an attacker from social media, and as such provide a very weak level of authentication.
```

Comment:
I believe this is repetitive with the introduction bit about `system defined` questions. The only spot where this has occurred.

Reply:
Fixed in 9ca5276